packages/components/ConfigModal/ConfigForm.tsx

```diff
@@ -0,0 +1,46 @@
+import { FormLabel, Input } from '@chakra-ui/react';
+
+import {
+  useConfigActions,
+  useConfigStates,
+} from '@packages/features/config-context';
+import { IConfigForm } from '@packages/entities/config-modal';
+
+export default function ConfigForm({ initialRef }: IConfigForm) {
+  const { onConfigChange } = useConfigActions();
+  const { electionDatabaseId, notionApiKey, resultsDatabaseId } =
```

Comment:
Idk why the fucking linter is breaking this line like this, but it is what it is, and I'm sad.